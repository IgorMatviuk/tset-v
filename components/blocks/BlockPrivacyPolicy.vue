<template>
  <div>
    <BContainer>
      <div class="privacy-policy">
        <h1 class="privacy-policy__title">{{ $t('pages.privacyPolicy') }}</h1>
        <hr class="privacy-policy__line" />
        <div v-html="$t('privacyPolicyPage')"></div>
      </div>
    </BContainer>
  </div>
</template>

<script>
export default {}
</script>

<style lang="scss">
.privacy-policy {
  padding-top: 52px;
  padding-bottom: 74px;

  h1 {
    font-size: 23px;
    line-height: 29px;
    color: $black;
    text-align: center;
    text-transform: uppercase;
  }

  h2 {
    margin-bottom: 12px;
    font-size: 11px;
    font-weight: 500;
    line-height: 14px;
    text-transform: uppercase;
  }

  h3 {
    margin-bottom: 13px;
    font-size: 11px;
    font-weight: 500;
    line-height: 14px;
  }

  p {
    font-size: 11px;
    line-height: 14px;
    color: $text-color;
  }

  &__line {
    margin: 17px 0;
    border-color: $primary-light-2;
  }

  @include media-breakpoint-up(md) {
    padding-top: 57px;
    padding-bottom: 98px;

    h1 {
      font-size: 30px;
      line-height: 38px;
    }

    &__line {
      margin-bottom: 23px;
    }

    h2 {
      margin-bottom: 15px;
      font-size: 12px;
      line-height: 15px;
    }

    h3 {
      font-size: 12px;
      line-height: 15px;
    }

    p {
      font-size: 12px;
      line-height: 15px;
    }
  }

  @include media-breakpoint-up(lg) {
    padding-bottom: 101px;

    @include make-col(10);
    @include make-col-offset(1);
  }

  @include media-breakpoint-up(xl) {
    padding-top: 54px;
    padding-bottom: 83px;

    h1 {
      font-size: 35px;
      line-height: 44px;
    }

    &__line {
      margin-bottom: 7px;
    }

    h2,
    h3,
    p {
      font-size: 14px;
      line-height: 18px;
    }

    h2 {
      padding-top: 16px;
    }

    h3 {
      margin-bottom: 10px;
    }
  }

  @include media-breakpoint-up(xxl) {
    padding-bottom: 100px;

    &__line {
      margin-bottom: 0;
    }

    h2 {
      padding-top: 22px;
    }
  }
}
</style>
