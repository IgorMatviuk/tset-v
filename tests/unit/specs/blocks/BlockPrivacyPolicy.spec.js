import { mount, createLocalVue } from '@vue/test-utils'

import { BContainer } from 'bootstrap-vue'

import BlockPrivacyPolicy from '@/components/blocks/BlockPrivacyPolicy'

const localVue = createLocalVue()
localVue.component('BContainer', BContainer)

describe('BlockPostNavigation', () => {
  function createBlockPrivacyPolicy() {
    const wrapper = mount(BlockPrivacyPolicy, {
      localVue,
      mocks: {
        $t: (path) => path
      }
    })

    return wrapper
  }

  test('render privacy policy block', () => {
    const wrapper = createBlockPrivacyPolicy()

    expect(wrapper.element).toMatchSnapshot()
  })
})
