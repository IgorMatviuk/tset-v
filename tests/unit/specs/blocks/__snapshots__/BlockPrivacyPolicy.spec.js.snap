// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlockPostNavigation render privacy policy block 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="privacy-policy"
    >
      <h1
        class="privacy-policy__title"
      >
        pages.privacyPolicy
      </h1>
       
      <hr
        class="privacy-policy__line"
      />
       
      <div>
        privacyPolicyPage
      </div>
    </div>
  </div>
</div>
`;
